orangecanvas/document/quickmenu.py

```diff
@@ -1602,6 +1555,14 @@ def exec_(self, pos=None, searchText=""):
         triggered. 'Search' text field is initialized with `searchText` if
         provided.
         """
+        # to avoid accidental category hovers, offset the quickmenu
+        # these were calculated by hand, the actual values can't be grabbed
+        # before showing the menu for the first time (and they're defined in qss)
+        x_offset = 33
+        if self.__pages.navigator.categoriesEnabled():
+            x_offset += 33
```

Comment:
This doesn't work well if the popup would stick outside the right margin of the screen - it is shown 33 px left of the mouse.

Reply:
Oh wow, nice catch.